docker-compose.yml

```diff
@@ -0,0 +1,59 @@
+version: '3.8'
+
+services:
+  app:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    container_name: gnosh-observer
+    depends_on:
+      - db
+      - prometheus
+    environment:
+      - DB_HOST=${DB_HOST}
+      - DB_PORT=${DB_PORT}
+      - DB_USER=${DB_USER}
+      - DB_PASSWORD=${DB_PASSWORD}
+      - DB_NAME=${DB_NAME}
+      - DB_SSL_MODE=${DB_SSL_MODE}
```

Comment:
AFAICS we don't need those to be configurable (with the exception of DB_PASSWORD maybe) from the "outside".